src/Lean/Parser/Basic.lean

```diff
@@ -1610,7 +1610,9 @@ inductive LeadingIdentBehavior where
   The method `termParser prec` is equivalent to the method above.
 -/
 structure ParserCategory where
-  tables   : PrattParsingTables
+  ref      : Name
```

Comment:
Could you please add doc strings? :)

Reply:
BTW, I didn't read the code carefully, but it seems `ref` is the `declName`, right?

Reply:
Yes. Shall I rename it to that? (I used `ref` for the attribute since the reference itself was mostly a dummy to access the location and doc string information, and that reasoning also applies here, although for categories there is a clearer relationship to a dedicated dummy definition of type `Lean.Parser.Category`, so `declName` also makes sense as the name.)

Reply:
For built-in parsers `ref` will be the `builtin_initialize` instead of the `Category` declaration, right? But I don't mind `declName` either.